src/mcl_3dl.cpp

```diff
@@ -585,10 +598,20 @@ class MCL3dlNode
       return;
     }
 
-    if (!pcl_ros::transformPointCloud(
+    try
+    {
+      if (!pcl_ros::transformPointCloud(
             frame_ids_["base_link"], *pc_local_accum_, *pc_local_accum_, tfl_))
+      {
+        ROS_INFO("Failed to transform pointcloud.");
```

Comment:
This error never happens since transform to `frame_ids_["odom"]` is already satisfied.